client/src/components/AdminProfile/PermissionsRadioGroup.js

```diff
@@ -0,0 +1,249 @@
+    }
+
+    li::marker {
+      color: ${colors.royalBlue} !important;
+    }
+  }
+
+  .ant-radio-wrapper .permissions {
+    max-height: 0;
+    overflow: hidden;
+  }
+
+  .ant-radio-wrapper-checked {
+    .radio-title::after {
+      content: " -";
+      font-size: 2rem;
+      font-weight: 500;
+    }
+
+    h3 {
+      margin-bottom: 3rem !important;
+    }
+
+    .permissions {
+      max-height: 100%;
+    }
+  }
+
+  .list-items {
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+  }
+
+  .ant-radio-checked::after {
+    display: none;
+  }
+`;
+
+export const PermissionsApplyButton = styled(BaseButton)`
+  border-radius: 4.6rem;
+  background-color: ${colors.royalBlue};
+  color: ${colors.white} !important;
+  font-family: ${typography.display};
+  font-weight: 500;
+  font-size: 1.6rem;
+  display: block;
+  max-width: 33.4rem;
+  width: 33.4rem;
+  height: 5.4rem;
+  margin: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    font-size: 1.4rem;
+    width: 15.5rem;
+    height: 4.8rem;
+  }
+`;
+
+const PermissionsRadioGroup = ({ permissions, loading }) => {
+
+  return (
+    <>
+      <RadioContainer
+        onChange={(event) => event.target.value}
+        defaultValue="Volunteer"
+      >
+        <Radio className="option" value={permissions} key="volunteer">
+          <div className="radio-content">
+            <h3 className="radio-title">Volunteer</h3>
+
+            <ul className="permissions">
+              <h4>Role Permisions:</h4>
+              <div className="list-items">
+                <li>Will be given a member badge</li>
+                <li>Can see org members’ permissions</li>
+              </div>
+            </ul>
+          </div>
+        </Radio>
+
+        <Radio className="option" value="orgwiki" key="orgwiki">
+          <div className="radio-content">
+            <h3 className="radio-title">Org Wiki Editor</h3>
+
+            <ul className="permissions">
+              <h4>Role Permisions:</h4>
+              <div className="list-items">
+                {
+                  // TODO - need role/permission text
+                }
+                <li>Lorem ipsum dolor si libero reiciendis quo non.</li>
+                <li>
+                  Deleniti sunt, soluta ea laboriosam, necessitatibus nisi ad,
+                  obcaecati.
+                </li>
+              </div>
+            </ul>
+          </div>
+        </Radio>
+
+        <Radio className="option" value="administrator" key="administrator">
+          <div className="radio-content">
+            <h3 className="radio-title">Administrator</h3>
+
+            <ul className="permissions">
+              <h4>Role Permisions:</h4>
+              <div className="list-items">
+                {
+                  // TODO - need role/permission text
+                }
+                <li>Lorem ipsum dolor si.</li>
```

Comment:
Same lorem ipsum. remove all of it and replace with actual text

Reply:
This is from previous deployment.   Some functionality on the join org feature is for future enhancement, which will only happen if FP org allows.